documentation/modules/ROOT/pages/transformations/mongodb-outbox-event-router.adoc

```diff
@@ -0,0 +1,430 @@
+:page-aliases: configuration/mongodb-outbox-event-router.adoc
+// Category: debezium-using
+// Type: assembly
+// ModuleID: configuring-debezium-connectors-to-use-the-outbox-pattern
+// Title: Configuring {prodname} connectors to use the outbox pattern
+[id="mongodb-outbox-event-router"]
+= MongoDB Outbox Event Router
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+[NOTE]
```

Comment:
Let's add a note about this SMT being under incubation, similar to this one [here](https://debezium.io/documentation/reference/stable/transformations/content-based-routing.html).